[2023-09-04T18:45:56.989+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: producer.update_dataset_2 scheduled__2023-09-03T00:00:00+00:00 [queued]>
[2023-09-04T18:45:57.007+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: producer.update_dataset_2 scheduled__2023-09-03T00:00:00+00:00 [queued]>
[2023-09-04T18:45:57.007+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-09-04T18:45:57.008+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-09-04T18:45:57.009+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-09-04T18:45:57.089+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): update_dataset_2> on 2023-09-03 00:00:00+00:00
[2023-09-04T18:45:57.103+0000] {standard_task_runner.py:55} INFO - Started process 87 to run task
[2023-09-04T18:45:57.143+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'producer', 'update_dataset_2', 'scheduled__2023-09-03T00:00:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/producer.py', '--cfg-path', '/tmp/tmplzjjyq31']
[2023-09-04T18:45:57.161+0000] {standard_task_runner.py:83} INFO - Job 57: Subtask update_dataset_2
[2023-09-04T18:45:57.369+0000] {task_command.py:376} INFO - Running <TaskInstance: producer.update_dataset_2 scheduled__2023-09-03T00:00:00+00:00 [running]> on host 993b415b7ab0
[2023-09-04T18:45:57.476+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=producer
AIRFLOW_CTX_TASK_ID=update_dataset_2
AIRFLOW_CTX_EXECUTION_DATE=2023-09-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-03T00:00:00+00:00
[2023-09-04T18:45:57.478+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-09-04T18:45:57.486+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/baseoperator.py:1390: RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
  context['ti'].xcom_push(key=key, value=value, execution_date=execution_date)

[2023-09-04T18:45:57.523+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=producer, task_id=update_dataset_2, execution_date=20230903T000000, start_date=20230904T184556, end_date=20230904T184557
[2023-09-04T18:45:57.588+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-09-04T18:45:57.614+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
